docker/rucio-consistency/Dockerfile

```diff
@@ -32,6 +32,10 @@ RUN yum install -y osg-ca-certs\
     && yum clean all \
     && rm -rf /var/cache/yum
 
+RUN mkdir -p /etc/grid-security \
+    && rm -rf /etc/grid-security/certificates \
+    && ln -s /cvmfs/grid.cern.ch/etc/grid-security/certificates /etc/grid-security/
+
```

Comment:
If you are doing this, you can for sure get rid of the osg-certificate install above